### ISSUE TITLE
check_usr_local_bin_pos.sh seems to give false negatives

### ISSUE THREAD

**Issue by TheIronBorn** [kristovatlas/osx-config-check#179]
my $PATH: 
```shell
/opt/local/bin:/opt/local/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/X11/bin
                                usr/local/bin:/usr/bin
```

yet the script outputs `0`
